community/gcovr/APKBUILD

```diff
@@ -1,30 +1,32 @@
 # Contributor: André Klitzing <aklitzing@gmail.com>
 # Maintainer: André Klitzing <aklitzing@gmail.com>
 pkgname=gcovr
-pkgver=3.4
+pkgver=4.0
 pkgrel=0
 pkgdesc="Generates a simple report that summarizes the gcc code coverage"
 url="http://gcovr.com/"
 arch="noarch"
 license="BSD"
-depends="python2"
-makedepends="python2-dev py-setuptools"
+depends="python3 py3-jinja2"
+makedepends="python3-dev py3-setuptools"
+checkdepends="py3-pytest"
+options="!check" # ModuleNotFoundError: No module named 'atomicwrites'
 source="gcovr-${pkgver}.tar.gz::https://github.com/gcovr/gcovr/archive/${pkgver}.tar.gz"
 builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
 	cd "$builddir"
-	python2 setup.py build
+	python setup.py build
```

Comment:
You have to use `python3` instead of `python` here.

Reply:
Thanks!